src/main/java/luke/Luke.java

```diff
@@ -0,0 +1,69 @@
+package luke;
+
+import luke.command.Command;
+
+import java.io.IOException;
+
+/**
+ * A <code>Luke</code> object is used to run the entire Luke application.
+ * It contains a <code>Ui</code> object, a <code>TaskList</code> object, a <code>Storage</code>, and a
+ * <code>Parser</code> object to execute various commands from the user.
+ */
+public class Luke {
+    /** An object to handle the user interaction with Luke */
```

Comment:
Good job writing descriptive header comments, and following proper layout for comments relative to their position in code